website/content/docs/what-is-boundary.mdx

```diff
@@ -6,56 +6,98 @@ description: And introduction to Boundary
+- "Just-in-time" network and credential access for sessions via [HashiCorp Vault](https://vaultproject.io/)
 - Single sign-on to target services and applications via external identity providers
 - Access-as-code to automate the configuration of user permissions
 - Automated discovery of target systems
 - Session monitoring and management for access created via Boundary.
 
-## Boundary Goals
-
-**Zero Trust Security**: The foundation of Boundary is an identity-based, “Zero-Trust”, access model. Zero-Trust means user access is continuously authenticated and only authorized when rules and policies tied to the user’s identity are verified.
+Boundary's foundation is based on the following important concepts.
 
-**A Consistent Workflow for Access**: Boundary connects the user securely to their infrastructure regardless of cloud platform, target environment, or identity provider. Boundary creates a consistent workflow for user authentication and authorization to create secure, ephemeral sessions that can be monitored and managed by administrators.
+- **Zero Trust Security:** Zero-Trust is an identity-based access model where the user access is continuously authenticated. Access is only authorized when the established rules and policies tied to the user’s identity are verified.
+- **Consistent Workflow for Access:** Once the user is verified and granted access, Boundary securetly connects the user securely to their infrastructure regardless of cloud platform, target environment, or identity provider. This foundation provides continuous user authentication and authorization workflows within ephemeral sessions, which administrators can monitor and manage securely.
+- **Extensibility with the Ecosystem:** Modern organizations often require a multilayered access matrix constructed of identity providers, policy engines, secrets management tools, target types, and cloud providers that integrate and allow users to reside within access workflows requiring vendor lock-in. Boundary does not require vendor lock-in and supports the user's vendor-of-choice.
 
 ![Boundary workflow](/img/boundary.png)
 
-**Extensibility with the Ecosystem**: Modern organizations have a multilayered matrix of identity providers, policy engines, secrets management tools, target types, and cloud providers that must integrate and reside within the access workflow. Boundary's extensible ecosystem supports the user's vendor-of-choice in this access workflow.
+## How does Boundary work?
+
+Boundary provides secure access to hosts and critical systems without distributing and managing credentidals, configuring firewalls, or exposing the organization's private network. Traditionally, for users to access their resources, it's required that organizations establish and maintain SSH bastion hosts and VPNs. The illustration below displays Boundary's core workflow.
+
+![Boundary core workflow](/img/boundary-core-workflow.png)
```

Comment:
Image seems blurry in the preview

Reply:
I'll see about fixing that image to make it clearer.

Reply:
I played around with the image size and font. It won't be as clear as the image above it (different tools used) but hopefully it's readable and a tad bit better. Eventually, we may need the assistance of the marketing team to fine-tune some of the images. Long term, a new writer should be able to get in there and fix some of them or at least work with the marketing folks to help fine-tune.